sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -138,6 +143,17 @@ struct work_group_size_hint_key {
   using value_t = property_value<work_group_size_hint_key, std::integral_constant<size_t, Dims>...>;
 }; // work_group_size_hint_key
 
+// Corresponds to max_work_group_size
```

Comment:
It is not clear which `max_work_group_size` is meant here. There is a device info query like this in the core SYCL 2020 spec and there is `intel::max_work_group_size` attribute which is not covered by any SYCL extension specifications AFAIK. Do we really need this comment?

Reply:
No I don't think so. As you say, it's an unspecified attribute so isn't helpful to point to it.